test/util.js

```diff
@@ -80,6 +82,41 @@ function createSubstitutionPostRequest(base_id, overlay_id, base_file, overlay_f
   return (reqParams);
 };
 
+function getYamlCmd(metadata) {
```

Comment:
_Isn't this basically a re-implementation of the command creation in the server?_

This looks a bit complex, so you now have two functions in the code constructing the command. What if you made the same mistake in this function again?

I suggest for a test to manually create the expected command and compare against that, only replacing the compendium ID.